nixos/modules/services/hardware/monado.nix

```
@@ -29,6 +29,18 @@ in
 
     highPriority = mkEnableOption "high priority capability for monado-service"
       // mkOption { default = true; };
+
+    environment = mkOption {
+      type = types.attrsOf types.str;
```

Maybe we can use the same type as systemd's `environment` option

```suggestion
      type = with types; attrsOf (nullOr (oneOf [ str path package ]));
```

---

What is the purpose of nullOr? Also why path and package? As far as I can see in the Monado env vars, only strings and numbers are used

---

It is just what systemd uses for `environment`. Note that you can any arbitrary env var here, including ones that are not directly consumed by Monado. So having all of these makes sense for those.

---

Okay, that makes sense, I'll change it